app/jobs/pubsub_job.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class PubsubJob < ApplicationJob
+  # self.queue_adapter = :pubsub
+  queue_as(:default)
+  retry_on(StandardError, wait: 5.minute, attempts: 2)
+
+  def perform(*args)
+    puts("\n [PubsubJob][perform-start] #{args}")
```

Comment:
Can this be addressed using instrumentation?

Reply:
Yes, I added the commit https://github.com/skysemilla/kisi-api-challenge/pull/1/commits/8425060cc4cd4472de51ee68f8080f3ed8d6d764 to implement the active support notification to instrument and subscribe to an event.